docs/detections/query-alert-indices.asciidoc

```diff
@@ -1,6 +1,6 @@
 [[query-alert-indices]]
-== Query alert indices
-This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the <<alert-schema, Alert schema>>.
+== Manage and query alert indices
+This page explains how you should manage and query alert indices, for example when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the <<alert-schema, Alert schema>>.
```

Comment:
```suggestion
This page explains how you should manage and query alert indices, for example, when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the <<alert-schema, Alert schema>>.
```

Reply:
Adding an extra comment because some of the diff lines are grayed out. 

Line 10: 
> This alias includes both the legacy (before 8.0.0) and the new alert indices.

Remove `both`.